.github/workflows/publish-dist.yml

```diff
@@ -29,9 +29,25 @@ jobs:
       - name: Commit to dist repo
         run: |
           set -ex
+          mkdir dist-repo
           cd dist-repo
+          TARS=(../dist-unpack/cockpit-*.tar.xz)
+          if [ "${#TARS[@]}" -ne 1 ]; then
+              echo "Expected exactly one tarball" >&2
+              exit 1;
+          fi
+          tar --strip-components=1 -xf "${TARS[0]}"
```

Comment:
I actually preferred when you explicitly listed the files to extract here...

Reply:
I never did actually, as that's so darn hard with tar. (I had a prediction of the stripped path prefix in test/make_dist.py at some point, maybe you are thinking of that?)

Reply:
Indeed, I'm not sure where I saw this anymore.  Too many rounds of looking at similar-but-different patches here.

I think I mentioned before: armed with the tarball name, you can efficiently and reliably determine the distdir, if you wanted to...

If we were going to make any change, though, I'd put back the `--exclude '.git*'`.  That was a really nice thing, and it would work here again, since we remove the `.zip` danger again by having a separate subdir.

You could also go full-paranoid (as I do in node_modules) and keep the gitdir separate, but that's really a matter of taste.